frame/composable-traits/src/nft/mod.rs

```diff
@@ -68,17 +68,17 @@ pub trait ReferenceNft<AccountId>:
 	// `who` is owner of NFT in case of reference NFT is reported burn (so owner is lost), it is
 	// auctioned
 	fn reference_mint_into<NFTProvider, NFT>(
-		_class: &Self::ClassId,
-		_instance: &Self::InstanceId,
+		_class: &Self::CollectionId,
```

Comment:
i ok with rename, just hope it is no conflict with any other stuff

Reply:
The rename is comming from the polkadot upgrade, not from us